File: Backend/src/main/java/com/app/zakat/util/ZakatUtil.java
package com.app.zakat.util;

import java.time.LocalDate;
import java.time.chrono.ChronoLocalDate;
import java.time.chrono.HijrahChronology;
import java.time.chrono.HijrahDate;

public class ZakatUtil {

	public static String usingHijrahChronology(LocalDate gregorianDate) {
	    HijrahChronology hijrahChronology = HijrahChronology.INSTANCE;
	    ChronoLocalDate hijriChronoLocalDate = hijrahChronology.date(gregorianDate);
	    return HijrahDate.from(hijriChronoLocalDate).toString();
	}
}

File: Backend/Application/Utils/ZakatUtil.cs
using System;
using System.Globalization;

namespace Application.Utils;

public class ZakatUtil
{
	public static string UsingHijrahChronology(DateTime gregorianDate)
	{
		CultureInfo cultureInfo = new CultureInfo("ar-SA");
		return gregorianDate.ToString("yyyy/MM/dd", cultureInfo);
	}
}
